src/examples/chrome/ChromeDefault.tsx

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Button } from '@zendeskgarden/react-buttons';
+import { XXL, MD } from '@zendeskgarden/react-typography';
+import { ReactComponent as ProductIcon } from '@zendeskgarden/svg-icons/src/26/relationshape-support.svg';
+import { ReactComponent as HomeIcon } from '@zendeskgarden/svg-icons/src/26/home-fill.svg';
+import { ReactComponent as EmailIcon } from '@zendeskgarden/svg-icons/src/26/email-fill.svg';
+import { ReactComponent as SettingsIcon } from '@zendeskgarden/svg-icons/src/26/settings-fill.svg';
+import { ReactComponent as ZendeskIcon } from '@zendeskgarden/svg-icons/src/26/zendesk.svg';
+import { ReactComponent as MenuTrayIcon } from '@zendeskgarden/svg-icons/src/16/grid-2x2-stroke.svg';
+import { ReactComponent as PersonIcon } from '@zendeskgarden/svg-icons/src/16/user-solo-stroke.svg';
+import {
+  Chrome,
+  Body,
+  Content,
+  Main,
+  Header,
+  HeaderItem,
+  HeaderItemIcon,
+  HeaderItemText,
+  Footer,
+  FooterItem,
+  Nav,
+  NavItem,
+  NavItemIcon,
+  NavItemText,
+  Sidebar
+} from '@zendeskgarden/react-chrome';
+
+const Example = () => (
+  <Row>
+    <Col>
+      <Chrome isFluid style={{ height: 500 }}>
```

Comment:
Also specify a `min-width` style (600?) to prevent the layout from squishing on mobile.